tree/dataframe/src/RDFInternalUtils.cxx

```diff
@@ -0,0 +1,24 @@
+// Author: Vincenzo Eduardo Padulano CERN 02/2023
+
+/*************************************************************************
+ * Copyright (C) 1995-2023, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#include "ROOT/RDF/InternalUtils.hxx"
+
+namespace ROOT {
+namespace Internal {
+namespace RDF {
+
+void ChangeEmptyEntryRange(ROOT::RDF::RNode node, std::pair<ULong64_t, ULong64_t> &&newRange)
```

Comment:
It's better to use at least

```suggestion
void RDF::ChangeEmptyEntryRange(ROOT::RDF::RNode node, std::pair<ULong64_t, ULong64_t> &&newRange)
```
Otherwise header and source can disagree on the parameters, because this here serves as a declaration that does not need to match the declaration in the header. With the namespace prefix it is required to match an existing declaration.

Reply:
This gives me
```
error: explicit qualification in declaration of ‘void ROOT::Internal::RDF::ChangeEmptyEntryRange
```

Reply:
That means that either the declaration from the header isn't visible here, or that declaration and implementation doesn't match (i.e. *exactly* the reason I was proposing this change). Did one of them have `RNode` and the other `const RNode &`? :-)